images/template-service-broker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# This image will run the template service broker.
+# The entry point name is not yet final.
+
+FROM openshift/origin
+
+ENTRYPOINT ["/usr/bin/openshift start template-service-broker"]
```

Comment:
You want to build it and push it? `hack/build-images.sh` and `hack/push-images.sh` should be all